src/matlab2tikz.m

```diff
@@ -767,6 +768,9 @@ function addCustomCode(fid, before, code, after)
             % legends refer to the specgraph.errorbarseries
             % handle which is 'Parent' to the line handle.
             k = find(get(child,'Parent') == plotChildren);
+            if(~isempty(k))
```

Comment:
I think we should put a comment here to clarify what a "group" is in this context. Some variation on 

> MATLAB bodeplots consist of groups of two lines (Magnitude, Phase). The lines have no 'DisplayName' but the containing Group has one.

would be fine.

Also just as a tiny remark: this statement can be written more concisely:
```matlab
isGroup = ~isempty(k);
```